docs/journalist.rst

```diff
@@ -469,45 +464,61 @@ choose **Wipe** to delete them.
-reboot, the malware will be gone.
-
-It is crucial, however, that you have a strategy for dealing with malware before
-you move documents off the *Secure Viewing Station* in electronic form.
-SecureDrop does not scan for or automatically remove malware. If you copy an
-original file you received via SecureDrop to your everyday workstation, and that
-file contains malware, you may still end up running the malware on your everyday
-workstation.
-
-For this reason, we recommend taking additional precautions. Printing a file
-is often safer than exchanging it electronically, and it has the additional
-benefits of removing embedded metadata (except for printer codes, watermarks,
-or similar identifiers that may not be visible to the naked eye).
-
-Alternatively, you can use the tools provided within Tails to examine
-documents, or convert files from one format to another (e.g., export a Word
-document as PDF). Fully mitigating the risks of malware is beyond the scope
-of this documentation.
+As long as you are using the latest version of Tails on your *Secure Viewing Station*,
+you can open submitted documents with a low risk of malicious files compromising
+it. Tails only preserves changes to your persistent volume across reboots,
+making it difficult for malware to take hold. The air-gap prevents malware
+from "phoning home". Keeping your *Secure Viewing Station* up-to-date is the
+single best way to protect its security.
+
+If you copy files you have received via SecureDrop to your everyday workstation
+and open them there in their original form, the risks from malware are
+significantly greater. SecureDrop does not scan for or remove malware, and your
+everyday workstation likely has a much greater `attack surface <https://en.wikipedia.org/wiki/Attack_surface>`__
+than the *Secure Viewing Station*. Malware running on your everyday workstation
+could exfiltrate information to a remote server, destroy data, or attempt to
+spread itself to other computers in your corporate network.
+
+Instead of exporting files in digital form, print them whenever possible. Printing
+essentially neutralizes malware, and it has the additional benefit of removing
+embedded metadata. Printing a document may, however, preserve watermarks,
+printer codes, steganographically encoded data, or other information
+not visible to the naked eye.
+
+If you must copy files in digital form (e.g., because of the quantity of
+information or the format), we recommend consulting with your administrator or
+your digital security staff before the export. Fully mitigating the risks of
+malware is beyond the scope of this documentation.
+
+.. tip::
+
+  Tails provides both graphical and command-line utilities that can be used to
+  convert files across formats (e.g., PDF to PNG). This can be a good way to
+  dramatically reduce the risk of malware, especially when converting to the
+  simplest possible format that can hold the information you need.
+
```

Comment:
I found the beginning of this section a little bit confusing.  What about something like:

SecureDrop itself does not protect you from receiving malicious submissions, but there are steps you can take to minimize the risk of malware. 

Using an up-to-date version of Tails on your Secure Viewing Station is the safest way to open and assess the documents you receive. Tails OS has features that offer more protection against compromise than your everyday computer, and viewing documents on an air-gapped system stops potential malware from "phoning home." 

Whenever possible, print hard copies of documents from the Secure Viewing Station instead of exporting them in digital form. This stops you from transferring potentially malicious files, as well as limits metadata from those files. (Note that printing a document may still preserve watermarks, printer codes, steganographically encoded data, or other information not visible to the naked eye.)

If you must copy files in digital form (e.g., because of the format or quantity of information), we recommend consulting with your administrator or your digital security staff beforehand. Fully mitigating the risks of malware is beyond the scope of this documentation.

..tip:: Converting files to simpler formats (e.g., PDF to PNG) can help reduce the risk of malware. Tails provides both graphical and command-line utilities for this.

Reply:
The changes look great! In my opinion the list is much clearer.